index.js

```diff
@@ -3,6 +3,8 @@
 const cwd = process.cwd();
 process.chdir(__dirname); // needed for global bin to find libraries
 
+const Mixpanel = require('mixpanel');
+const mixpanel = Mixpanel.init('7c1cbc95d6519b7d6417d6fe1ce393e3');
```

Comment:
Probably would:
- break this code out to a text file
- require it much lower in the booting, so it's not loaded in the module case and close to bugsnag
- have a flag to disable it

Reply:
flag added at ff8fdaf
moved require at  3f427dd
load key at 608d3ba